runtime/builtin_types_test.go

```diff
@@ -23,6 +23,32 @@ import (
 	"testing"
 )
 
+func TestBuiltinDelAttr(t *testing.T) {
+	f := NewRootFrame()
+	delattr := mustNotRaise(Builtins.GetItemString(NewRootFrame(), "delattr"))
```

Comment:
`Builtins.GetItemString(NewRootFrame(), ...)` can be `Builtins.GetItemString(f, ...)`. You can use a frame more than once. If it raises then you need to `f.RestoreExc(nil, nil)`, but mustNotRaise() ensures this is not raising.